.env-example

```diff
@@ -30,3 +30,4 @@ DEFAQTO_API_TIMEOUT=
 MONGO_HOSTNAME=
 PACS_MONGOD_HOST='http://localhost:27017/'
 PACS_MONGOD_DB='frontend_development'
+PACS_CACHE_ENABLED=false
```

Comment:
I am pretty sure is on your radar but just in case ... don't forget to add the var on scripts repo. ;)

Reply:
Yeah, it is indeed. Will have a PR later on, thanks for reminding it anyway. 🙇